docs/manual/external-tools.tex

```diff
@@ -990,7 +980,26 @@
   Error Prone \<javac.jar>.  They were declared as Maven dependencies above.
```

Comment:
`They were declared as Maven dependencies above.`. That's no longer true.  Maybe just delete this?